pmix_governance.tex

```diff
@@ -320,6 +320,24 @@ \subsubsection{General voting rules}\label{general-voting-rules}}
   as having been voted \emph{in absentia} in the official tally.
 \end{itemize}
 
+Revision Exception votes are special votes for PRs that have been modified
+after the two-week review period deadline before a quarterly meeting 
+before a first vote or after having passed a first
+vote. Revision Exception votes can be called by the proposer to evaluate 
+whether the ASC is willing to proceed voting on the PR including the 
+modifications. 
+The idea behind Revision Exception votes is to maintain forward progress on
+PRs even if changes are made after the quarterly meeting deadlines or after
+a first vote.
+Normally, changes acceptable for Revision Exception votes
+are relatively minor and do not change the fundamentals of the PR. However,
+if all members of the ASC eligible to vote and in attendance at a quarterly
+meeting vote affirmatively for a Revision Exception vote, then the regular vote 
+for the PR can proceed for the text of the PR including the modifications. 
+If any member of the ASC eligible to vote votes negatively for the Revision
+Exception vote, the proposer must chose to re-read the PR with the modifications
+or to continue the normal vote without the modifications.
+
```

Comment:
This misses the case of abstain votes - does a revision vote with all but one affirmative vote pass? Does a revision vote with one affirmative vote and the rest abstains pass? I think it would be good to allow abstains, but add a quorum. Otherwise, even one person being out if the room at the time of the vote will derail the exceptional vote.

Reply:
Maybe something like:
 > A quorum of ASC voting-eligible members must be present and vote in the affirmative without any negative votes for the Revision Exception vote to pass.

Earlier in the document, we define what a quorum means.

Reply:
I fixed this with Josh's suggestion. Thanks!